docker-compose.override.yml

```diff
@@ -12,7 +12,7 @@ services:
       target: development
     volumes:
       - ./webapp/src:/service/webapp/src:ro
-      - ./packages/pong-engine:/service/packages/pong-engine:ro
+      - ./packages:/service/packages:ro
```

Comment:
```suggestion
```

suggestion: We could remove this since we need to rebuild the images if we change a package.

Same suggestion for the transcendence-app volume.